Source/WebCore/svg/graphics/SVGImageCache.cpp

```diff
@@ -57,6 +63,9 @@ void SVGImageCache::setContainerContextForClient(const CachedImageClient& client
     FloatSize containerSizeWithoutZoom(containerSize);
     containerSizeWithoutZoom.scale(1 / containerZoom);
 
+    if (zoomedAndRoundedContainerSizeIsEmpty(containerSizeWithoutZoom, containerZoom))
+        return;
```

Comment:
Why do we use `containerSizeWithoutZoom` and `containerZoom` instead of using `containerSize`? `containerSizeWithoutZoom` is calculating by dividing `containerSize` by `containerZoom`. I would suggest using:

```
    if (flooredIntSize(containerSize).isEmpty())
        return;
```

Reply:
I was thinking that the round trip of un-scaling, then re-scaling could introduce rounding artifacts that might reveal an empty container in cases where simply avoiding the round trip would not. But it's probably not likely and your approach is simpler.